notebooks/supernova_model_sims.ipynb

```diff
@@ -9,72 +9,298 @@
    "source": [
```

Comment:
<p>I guess the x-axis Phase is in units of days? Can you add the proper units?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>